components/FMenu.vue

```diff
@@ -190,6 +193,10 @@ export interface FMenuProps {
    * Disable the menu
    */
   disabled?: boolean;
+  /**
+   * Disable fade transitions
+   */
+  static?: boolean;
```

Comment:
`static` can be confused with other meanings. Actually, even some people define _static animations_, in regard of _dynamic animations_. So this si not a good choice, I would maybe use :
```suggestion
  inanimated?: boolean;
```